src/main/java/org/broadinstitute/hellbender/tools/spark/pipelines/ReadsPipelineSpark.java

```diff
@@ -112,6 +112,9 @@
     @Argument(doc = "the join strategy for reference bases and known variants", fullName = "join-strategy", optional = true)
     private JoinStrategy joinStrategy = JoinStrategy.BROADCAST;
 
+    @Argument(fullName = MarkDuplicatesSpark.DO_NOT_MARK_UNMAPPED_MATES, doc = "Enabling this option will mean unmapped mates of duplicate marked reads will not be marked as duplicates.")
```

Comment:
this should probably be an advanced argument or a not recommend one or something

Reply:
I mean, how "advanced" is matching gatk3 vs not matching gatk?

Reply:
very advanced.  We need a "not recommended" argument option